modules/account-lib/package.json

```diff
@@ -41,6 +41,7 @@
     "@bitgo/sdk-coin-eth2": "^1.2.27",
     "@bitgo/sdk-coin-hash": "^1.1.0",
     "@bitgo/sdk-coin-hbar": "^1.3.27",
+    "@bitgo/sdk-coin-injective": "^1.1.2",
```

Comment:
Is this required in account-lib for other services or can we remove it?

Reply:
Yes this will be required for Wallet Platform.